### PR TITLE
PCM-849 - Log Channel id with streaming client errors

### PR DESCRIPTION
**Pull request by ckbrewer33**
Added channelId to error logs
updated changelog file to include topic combining fix that went out in v11.0.1 release

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -130,7 +130,8 @@ export default class Client {
     });
 
     this.on('sasl:failure', (err) => {
-      this.logger.error('Authentication failed connecting to streaming service', err);
+      const errMessage = `Authentication failed connecting to streaming service\nchannelId: ${this.config.channelId}`;
```
Comment (djhouseknecht) [inline src/client.js]: It looks like a minor typo in `service\nchannelId:` to `service\channelId:`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 #Changelog
+## v11.0.2
+* Included channelId in error logs
 
 ## v11.0.1
 * Added changelog file
+* Fixed an issue with topics not being combined correctly - [PR #43](https://github.com/purecloudlabs/purecloud-streaming-client/pull/43)
```
Comment (jensengar) [inline CHANGELOG.md]: thanks for putting this in.

File: src/client.js (hunk 1 above)
Comment (ckbrewer33) [reply src/client.js]: That's a newline character :)

Comment (djhouseknecht) [reply src/client.js]: Oof, you are right. For some reason I was reading it like you were trying to do a normal `\`

**Pull request merged**